docker/app/Dockerfile

```diff
@@ -3,10 +3,18 @@
 
 FROM python:3.7.3
 
+ENV DEBIAN_FRONTEND=noninteractive
+
 RUN apt-get update -y
-RUN apt-get install -y build-essential curl unzip man-db gcc
+RUN apt-get -y upgrade
```

Comment:
Is it required? This will create non-reproducible builds.

Reply:
What specifically will create non-reproducible builds? The `--ignore-missing` flag? That came from @dirkpetersen so let's ask him if it's necessary.

Reply:
Oh, I think you mean the `apt-get upgrade`. Again, that's from @dirkpetersen .

Reply:
Yeah, the `upgrade`, I do not subscribe to the idea of having the "latest" all the time. If necessary, let's leave it in and remove in the future as necessary, but I guarantee that it will cause a bug further down the line. 

Would it be possible to have the merge issues resolved. Then I can merge to master.